src/components/Header.js

```diff
@@ -1,12 +1,102 @@
+import {useEffect, useState} from 'react';
 import {Link} from 'react-router-dom';
+import styled from 'styled-components';
+
+import headerImg from './header.png';
 
 export default function Header() {
+	const [menu, setMenu] = useState(false);
+	function toggleMenu() {
+		setMenu(!menu);
+		console.log(menu);
+	}
+
 	return (
 		<header>
-			<nav>
-				<Link to="/">Home</Link>
-				<Link to="/about">About me</Link>
-			</nav>
+			<Navigation>
+				<Home>
+					<Link to="/">
+						<HeaderImage src={headerImg}></HeaderImage>
+					</Link>
+				</Home>
+				<DropDown>
+					<MenuWrapper>
+						<Menu>Navigation</Menu>
+						<Down onClick={toggleMenu}>Down</Down>
+					</MenuWrapper>
+
+					{menu && (
+						<Container>
+							<NavElement>
+								<Link to="/">Home</Link>
+							</NavElement>
+							<NavElement>
+								<Link to="/about">About me</Link>
+							</NavElement>
+						</Container>
+					)}
+				</DropDown>
+			</Navigation>
 		</header>
 	);
 }
+
+const Navigation = styled.nav``;
+
+const Home = styled.div`
```

Comment:
If you really need this `<div>` for styling purposes think about a more descriptive name like HomeLink, StyledHomeLink....

Reply:
Removed.